extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala

```diff
@@ -520,28 +520,50 @@ class HiveCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite {
     }
   }
 
-  test("[KYUUBI #3326] check persisted view and skip shadowed table") {
+  test("check persisted view and skip shadowed table") {
     val table = "hive_src"
     val permView = "perm_view"
     val db1 = "default"
     val db2 = "db2"
+    val userPermViewOnly = "user_perm_view_only"
 
     withCleanTmpResources(Seq(
       (s"$db1.$table", "table"),
       (s"$db2.$permView", "view"),
       (db2, "database"))) {
-      doAs("admin", sql(s"CREATE TABLE IF NOT EXISTS $db1.$table (id int)"))
+      doAs("admin", sql(s"CREATE TABLE IF NOT EXISTS $db1.$table (id int, name string)"))
 
       doAs("admin", sql(s"CREATE DATABASE IF NOT EXISTS $db2"))
-      doAs("admin", sql(s"CREATE VIEW $db2.$permView AS SELECT * FROM $table"))
+      doAs("admin", sql(s"CREATE VIEW $db2.$permView AS SELECT * FROM $db1.$table"))
 
+      // KYUUBI #3326: with no privileges to the permanent view or the source table
       val e1 = intercept[AccessControlException](
-        doAs("someone", sql(s"select * from $db2.$permView")).show(0))
+        doAs("someone", { sql(s"select * from $db2.$permView").collect() }))
       if (isSparkV31OrGreater) {
         assert(e1.getMessage.contains(s"does not have [select] privilege on [$db2/$permView/id]"))
       } else {
         assert(e1.getMessage.contains(s"does not have [select] privilege on [$db1/$table/id]"))
       }
+
+      // KYUUBI 4504: query the perm view
+      // with access privileges to the permanent view but no privilege to the source table
+      val sql1 = s"select * from $db2.$permView"
```

Comment:
wrap another test("")

Reply:
It's just another case of the same context. There's no gain to repeating building it in a separate test.
Multiple cases in the same ut is allowed like previous PR [#4532](https://github.com/apache/kyuubi/pull/4532/files#diff-c3edd338a81dae5890859798af29606cf51ff6a24c9c4e5fbc6190054de6f0beR1660).

Reply:
Addressed.